react-graph/src/Plot.tsx

```diff
@@ -60,7 +62,11 @@ export interface IProps {
     legendHeight?: number,
     legendWidth?: number,
     useMetricFactors?: boolean,
-    onSelect?: (t:number) => void,
+    // onSelects are functions guaranteed to run
```

Comment:
I am not sure we actually need this. I think the `onSelect` was originally here to figure out what happens when the user clicks the `Select` button - if `undefined` it will actually hide the button so the meaning of `onSelect` here does not match the meaning in the handler - which is ok because handlers are internal to `react-graph` but it means we do not need to change them here